include/boost/histogram/utility/wald_interval.hpp

```diff
@@ -29,8 +29,8 @@ class wald_interval : public binomial_proportion_interval<ValueType> {
   interval_type operator()(value_type successes, value_type failures) const noexcept {
     const value_type ns = successes, nf = failures;
     const value_type n = ns + nf;
-    const value_type a = (ns) / (n);
-    const value_type b = (z_ / (n) * std::sqrt(n)) * std::sqrt(ns * nf);
+    const double a = ns / n;
+    const double b = (z_ / n) * std::sqrt(ns * nf / n);
```

Comment:
use value_type not double, otherwise good

Reply:
Umm... But what if it's an int ?

Reply:
Got it nevermind.